client/utils/account-fees.tsx

```diff
@@ -74,28 +82,33 @@ const getFeeDescriptionString = ( fee ) => {
 	return '';
 };
 
-export const getCurrentFee = ( accountFees ) => {
+export const getCurrentFee = (
+	accountFees: FeeStructure
+): BaseFee | DiscountFee => {
 	return accountFees.discount.length
 		? accountFees.discount[ 0 ]
 		: accountFees.base;
 };
 
-export const formatMethodFeesTooltip = ( accountFees ) => {
+export const formatMethodFeesTooltip = (
+	accountFees: FeeStructure
+): JSX.Element | undefined => {
 	if ( ! accountFees ) return;
 
 	const total = {
 		percentage_rate:
 			accountFees.base.percentage_rate +
-			accountFees.additional.percentage_rate +
-			accountFees.fx.percentage_rate,
+			( accountFees.additional?.percentage_rate || 0 ) +
+			( accountFees.fx?.percentage_rate || 0 ),
 		fixed_rate:
 			accountFees.base.fixed_rate +
-			accountFees.additional.fixed_rate +
-			accountFees.fx.fixed_rate,
+			( accountFees.additional?.fixed_rate || 0 ) +
+			( accountFees.fx?.fixed_rate || 0 ),
```

Comment:
Looks fishy, are the definitions correct?

Reply:
Same as previous comment, I'll double check definitions